lib/livebook_web/live/home_live/session_list_component.ex

```diff
@@ -136,6 +136,13 @@ defmodule LivebookWeb.HomeLive.SessionListComponent do
               </button>
             </:toggle>
             <:content>
+              <a class="menu-item text-gray-500"
+                role="menuitem"
+                href={Routes.session_path(@socket, :download_source, session.id, "livemd", include_outputs: false)}
+                download>
+                <.remix_icon icon="download-2-line" class="text-lg"/>
+                <span class="font-medium">Download source</span>
+              </a>
```

Comment:
@josevalim any preference for the actions order? I think it's fine as the first one.

Reply:
I actually prefer the 2nd order, as I would expect a minor action (like download a source), before a destructive action like disconnecting the runtime or closing the session.

Reply:
@Benjamin-Philip I prefer it first because it mirrors the in notebook menu where export is first. :)

Reply:
When I refer to "2nd order", I mean the order you committed :). Plus there's also your (more valid) point.